omnigibson/prims/entity_prim.py

```diff
@@ -768,32 +689,14 @@ def _denormalize_efforts(self, efforts, indices=None):
         """
         return efforts * self.max_joint_efforts if indices is None else efforts * self.max_joint_efforts[indices]
 
-    def _verify_physics_view_is_valid(self):
```

Comment:
is this no longer needed? iirc it becomes invalid sometimes after a sim step is taken or so

Reply:
We need to dig a bit deeper into this. it only ever gets invalidated in the same cases where we need to re-call update_handles, so if it ever happens it means we need to put more calls to update_handles in places instead of fixing it this way.